scripts/generateDocs.js

```diff
@@ -199,28 +199,30 @@ function processApiExamples() {
 
 function copyReadmeLocally() {
 
-  if (!fs.existsSync(dirDocTemplates)){
-    fs.mkdirSync(dirDocTemplates);
-  }
+  // Temporarily disable readme template syncing
 
-  if (!fs.existsSync(readmeFilePath)) {
-    fs.writeFile(readmeFilePath, '', function(err) {
-      if(err) {
-        console.log(chalk.red('ERROR: Unable to create README.md file.', err));
-      }
-    });
-  }
+  // if (!fs.existsSync(dirDocTemplates)){
+  //   fs.mkdirSync(dirDocTemplates);
+  // }
 
-  https.get(readmeTemplateUrl, function(response) {
-    let writeTemplate = response.pipe(fs.createWriteStream(readmeFilePath));
+  // if (!fs.existsSync(readmeFilePath)) {
+  //   fs.writeFile(readmeFilePath, '', function(err) {
+  //     if(err) {
+  //       console.log(chalk.red('ERROR: Unable to create README.md file.', err));
+  //     }
+  //   });
+  // }
 
-    writeTemplate.on('finish', () => {
+  // https.get(readmeTemplateUrl, function(response) {
+  //   let writeTemplate = response.pipe(fs.createWriteStream(readmeFilePath));
+
+  //   writeTemplate.on('finish', () => {
       processReadme();
-    });
+  //   });
 
-  }).on('error', (err) => {
-    console.log(chalk.red('ERROR: Unable to fetch README.md file from server.', err));
-  });
+  // }).on('error', (err) => {
+  //   console.log(chalk.red('ERROR: Unable to fetch README.md file from server.', err));
+  // });
```

Comment:
Looks like you are bypassing the original function for a local update. Please delete this code.

Reply:
@blackfalcon I don't want to remove this code. Leaving it in place is a reminder that this code is currently running in a non-standard format that we don't want to support longer than necessary. 

This is a temporary change until we get the template in the generator repo updated per this ticket: https://github.com/AlaskaAirlines/WC-Generator/issues/352.

I'd like to actually prioritize getting ticket 352 completed.

Reply:
We have version control, there is no need to keep commented code around with an ambiguous intent to bring it back. The preferred method is to remove this code with a commit that clearly states that this is what happened.